heartbeat/status_beater.go

```diff
@@ -31,6 +33,9 @@ const (
 	ServiceStopped = 3
 )
 
+// fqBeatName is the fully qualified beat name
+var fqBeatName string
```

Comment:
You can get rid of this. Not needed.

Reply:
Change back to have status func logp fqBeatName as SIEM beats also log fqBeatName for heartbeat
See SIEM ./internal/pkg/heartbeat/status_beater.go func StatusBeater